src/towncrier/_writer.py

```diff
@@ -44,10 +44,7 @@ def append_to_newsfile(
         if header:
             f.write(header)
 
-        f.write(content)
-
-        if prev_body:
-            f.write(f"\n\n{prev_body}")
+        f.write(content + prev_body if prev_body else content.rstrip() + "\n")
```

Comment:
maybe add a comment why the `content.rstrip() + '\n'` vs `content + prev_body` 

I guess that this is here to support having separate news files for each release or for draft output.

------

Also, I was expecting for the writer, to just do the write,  without updating the content.

Generating the content should be the job of the builder.

Reply:
It's an interesting point about the builder vs writer responsibilities. Maybe the names aren't great, but my impresson is that `builder` is responsible for generating new content, whereas the writer is responsible for incorporating this new content in an existing news file.

Reply:
I have the same view :) `builder` renders new content. `writers` puts together the new rendered contents with the existing (already rendered) content :)